frontend/server/src/Controllers/Problem.php

```diff
@@ -5633,4 +5634,43 @@ public static function getLibinteractiveGenForSmarty(\OmegaUp\Request $r): array
             \OmegaUp\FileHandler::deleteDirRecursively($dirname);
         }
     }
+
+    /**
+     *
+     * @return array{smartyProperties: array{payload: ProblemCollectionDetails, title: \OmegaUp\TranslationString}, entrypoint: string}
+     *
+     * @omegaup-request-param string $collection_type
+     */
+    public static function getCollectionsDetailsForSmarty(\OmegaUp\Request $r): array {
+        \OmegaUp\Validators::validateStringNonEmpty(
+            $r['collection_type'],
+            'collection_type'
+        );
```

Comment:
Esta es la nueva forma de usar el validador:
```suggestion
        $collectionType = $r->ensureString('collection_type')
```
Y de paso, ya no necesitas mandarla llamar después, en vez de eso sólo usas la variable: `$collectionType`